### PR TITLE
A few more spelling fixes

### PR DESCRIPTION
**Pull request by hammerandtongs**
Thanks for the great book, it's a good read so far.

### CODE REVIEW

File: composite.rst [hunk 1]
```diff
@@ -280,9 +280,9 @@ The idea now is to write a function ``Older10`` that takes an array of 10
 
     //return the older person in a group of 10 persons
     func Older10(people [10]person) person{
-        older := people[0] //the first one is yje older for now
```
Comment (initpy) [inline composite.rst:283]: Wow! kudos on figuring out that "yje" is actually "the"! It takes a good 
knowledge of the keyboard to understand the typo! Thank you! :)

**Comment by initpy** [comment]
Thanks again for your help, Adam!

**Comment by hammerandtongs** [comment]
Thanks for the book!

Could you please redeploy with the fixes? I'll try reading for content again now that the spelling doesn't distract me ;)

**Comment by initpy** [comment]
Thank _you_ for your help, Adam!
I just deployed a new version, after merging your pull request and another good one by Wayne https://github.com/initpy/go-book/pull/6/files --You, guys, did a _huge_ work! I'm really grateful.

**Pull request merged**